active/gridsyntax/GridSyntaxSpec.md

```diff
@@ -0,0 +1,335 @@
+
+### Other Use Cases for Succinct Syntax
+The succinct syntax was created with the goal of making Grid more intuitive. However, this new syntax will be implemented as a language feature and work properly for any read-only collection-typed property. The current syntaxes will still be functional for all other scenarios, as it will be with Grid. Examples of some of these scenarios include, but are not limited to:
+#### CalendarView
+Current Syntax to populate SelectedDates collection:
+```xml
+calendarView1.SelectedDates.Add(new DateTime(1977, 1, 5));
+calendarView1.SelectedDates.Add(new DateTime(1997, 6, 19));
+```
+New, equally functional syntax using proposed XAML language feature:
+```xml 
+<CalendarView SelectedDates="'1/5/1977', '6/19/1997'" />
+```
+
+#### ListView
+Current Syntax to populate ListView Items collection:
+```xml
+<ListView>
+    <ListView.Items>
+        <x:string>Hello</x:string>
+        <x:string>World</x:string>
+    </ListView.Items>
+</ListView>
+```
+New, equally functional syntax using proposed XAML language feature:
+```xml 
+<ListView Items="'Hello', 'world'" />
+```
+
+### User Error
+If a developer uses both syntaxes in their Grid definition, i.e.
+```xml
+//Warning: Incorrect syntax
+<Grid ColumnDefinitions="1*, 2*, Auto, *, 300" RowDefinitions="1*, Auto, 25, 14, 20">
+    <Grid.ColumnDefinitions>
+          <ColumnDefinition Width="1*" />
+          <ColumnDefinition Width="2*" />
+          <ColumnDefinition Width="Auto" />
+          <ColumnDefinition Width="*" />
+          <ColumnDefinition Width="300" />
+    </Grid.ColumnDefinitions>
+    <Grid.RowDefinitions>
+          <RowDefinition Height="1*" />
+          <RowDefinition Height="Auto" />
+          <RowDefinition Height="25" />
+          <RowDefinition Height= "14" />
+          <RowDefinition Height="20" />
+    </Grid.RowDefinitions>
+</Grid>
+```
+a duplicate property error will be thrown. This is already implemented for read/write properties, but will be implemented for read-only properties such as these as well. 
+
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+**ColumnDefinition API**
+```csharp
+[contract(Windows.Foundation.UniversalApiContract, 1)]
```

Comment:
I don't think this is correct. We should follow up with @MikeHillberg off-line about our contract story.

Reply:
This is on the Width property, which is indeed UAC1.

Reply:
In WinUI3, we won't have the UAC contract